rs/nns/governance/src/governance.rs

```diff
@@ -584,160 +583,7 @@ impl NnsFunction {
                 | NnsFunction::RetireReplicaVersion
         )
     }
-}
```

Comment:
Note that we took the opportunity here to merge to `impl NnsFunction`s. I assume the split was an oversight resulting from "we just dump everything into rs/nns/governance/src/governance.rs". hash tag the code growed organically.